### PR TITLE
Super agent recipe final step check status endpoint

### PR DESCRIPTION
**Pull request by alvarocabanas**
Adapt the recipe to query (with retries) for a healthy status in the SA.

Rename the display name and logs as New Relic Agent Control

### CODE REVIEW

File: recipes/newrelic/infrastructure/super-agent/debian.yml [hunk 1]
```diff
@@ -574,16 +574,39 @@ install:
...
+              statusCheckOutput="{}"
+            fi
+            STATUS=$(echo $statusCheckOutput | /usr/local/bin/newrelic utils jq '.super_agent.healthy')
+            if [ "$STATUS" == "true" ]; then
+              echo "Agent status check ok."
+              break
+            else
+              if [ "$TRIES" -eq "$MAX_RETRIES" ]; then
+                # Process-based validation attempt
+                IS_AGENT_INSTALLED=$(sudo ps aux | grep newrelic-super-agent | grep -v grep | wc -l)
```
Comment (rubenruizdegauna) [inline recipes/newrelic/infrastructure/super-agent/debian.yml:598]: Not critical, but `sudo` is not required here ☝️

File: recipes/newrelic/infrastructure/super-agent/rhel.yml [hunk 2]
```diff
@@ -517,13 +517,36 @@ install:
...
+              statusCheckOutput="{}"
+            fi
+            STATUS=$(echo $statusCheckOutput | /usr/local/bin/newrelic utils jq '.super_agent.healthy')
+            if [ "$STATUS" == "true" ]; then
+              echo "Agent status check ok."
+              break
+            else
+              if [ "$TRIES" -eq "$MAX_RETRIES" ]; then
+                # Process-based validation attempt
+                IS_AGENT_INSTALLED=$(sudo ps aux | grep newrelic-super-agent | grep -v grep | wc -l)
```
Comment (rubenruizdegauna) [inline recipes/newrelic/infrastructure/super-agent/rhel.yml:538]: Not critical, but `sudo` is not required here ☝️

File: recipes/newrelic/infrastructure/super-agent/suse.yml [hunk 3]
```diff
@@ -466,13 +466,36 @@ install:
...
+              statusCheckOutput="{}"
+            fi
+            STATUS=$(echo $statusCheckOutput | /usr/local/bin/newrelic utils jq '.super_agent.healthy')
+            if [ "$STATUS" == "true" ]; then
+              echo "Agent status check ok."
+              break
+            else
+              if [ "$TRIES" -eq "$MAX_RETRIES" ]; then
+                # Process-based validation attempt
+                IS_AGENT_INSTALLED=$(sudo ps aux | grep newrelic-super-agent | grep -v grep | wc -l)
```
Comment (rubenruizdegauna) [inline recipes/newrelic/infrastructure/super-agent/suse.yml:487]: Not critical, but `sudo` is not required here ☝️

File: recipes/newrelic/infrastructure/super-agent/debian.yml (hunk 1 above)
Comment (alvarocabanas) [reply recipes/newrelic/infrastructure/super-agent/debian.yml:598]: Thanks for poiting it out, I'll modify it.

Comment (alvarocabanas) [reply recipes/newrelic/infrastructure/super-agent/debian.yml]: Fixed.

File: recipes/newrelic/infrastructure/super-agent/rhel.yml (hunk 2 above)
Comment (alvarocabanas) [reply recipes/newrelic/infrastructure/super-agent/rhel.yml]: Fixed.

File: recipes/newrelic/infrastructure/super-agent/suse.yml (hunk 3 above)
Comment (alvarocabanas) [reply recipes/newrelic/infrastructure/super-agent/suse.yml]: Fixed.

**Review by rubenruizdegauna — APPROVED** [review]
LGTM

**Review by NSSPKrishna — APPROVED** [review]
LGTM

**Pull request merged**